e2e/test/provisioning/ReprovisioningE2ETests.cs

```diff
@@ -936,9 +949,9 @@ public static void CleanupCertificates()
             {
                 s_x509CertificatesFolder.Delete(true);
             }
-            catch (Exception)
+            catch (Exception ex)
             {
-                // In case of an exception, silently exit. All systems images on Microsoft hosted agents will be cleaned up by the system.
+                Console.Write($"Client certificate cleanup failed due to {ex}");
```

Comment:
Logger?

Reply:
ClassCleanup is static and no access to the logger. :(

Reply:
That's a shame, but fair enough.